extensions/sql-migration/src/wizard/skuRecommendation/skuRecommendationPage.ts

```diff
@@ -434,9 +239,12 @@ export class SKURecommendationPage extends MigrationWizardPage {
 			level: azdata.window.MessageLevel.Error
 		};
 
+		this._serverName = this.migrationStateModel.serverName || (await getSourceConnectionProfile()).serverName;
```

Comment:
this will fine if this.migrationStateModel.serverName is null, but incase if this having an empty string what will be the behavior?

Reply:
In that case, it will display empty value in the places it is used. But it should not be the case since this was previously also there.